### DISCUSSION TITLE
Component proposal: Introducing charts component

### DISCUSSION THREAD

**Discussion by leonardoRocchini** [shadcn-ui/ui#1154] [Ideas]
Hi shadcn, thank you for what you have done for the community.
the integration of a new chart component would be great like the one in the themes section.
I think you used Recharts for do it right?
maybe you're probably already working on it.

what you guys think bout this?

**Comment by Samuele-Barbiera** [comment]
it would be great

**Comment by PabloResende** [comment]
😎😎😎😎

**Comment by petermekhaeil** [comment]
The example page is using [rechart](https://recharts.org/en-US/).

https://github.com/shadcn-ui/ui/blob/main/apps/www/registry/default/example/cards/stats.tsx

**Reply by HAYVENO** [reply]
Just what I needed.

**Comment by HAYVENO** [comment]
> Hi shadcn, thank you for what you have done for the community. the integration of a new chart component would be great like the one in the themes section. I think you used Recharts for do it right? maybe you're probably already working on it.
> 
> what you guys think bout this?

Came to request for this.

**Comment by MHRSRoni** [comment]
add a chart component.. it will be helpfull

**Comment by shadcn** [comment]
Shipped! https://ui.shadcn.com/charts

Thank you.

**Reply by abdullah-hallaq** [reply]
Hello,

Good work, but I have one suggestion. Couldn't you make them reusable? For example, make the bar chart accept some props, and it will map over them and display accordingly.
